composer.json

```diff
@@ -27,19 +27,21 @@
         "symfony/config": "~3.0|~4.0|~5.0",
         "symfony/http-kernel": "~3.0|~4.0|~5.0",
         "symfony/property-info": "^3.4.2|~4.0|~5.0",
-        "monolog/monolog": "~1.22",
+        "monolog/monolog": "^1.22|^2.0|^3.0",
         "php-http/client-common": "^2.3",
         "php-http/discovery": "^1.3",
         "php-http/httplug": "^2.2",
         "psr/http-message": "^1.0",
-        "psr/log": "^1.1"
+        "psr/log": "^1.1|^2.0|^3.0",
+        "php-http/message-factory": "^1.1"
     },
     "require-dev": {
-        "symfony/yaml": "~3.0|~4.0|~5.0",
-        "symfony/console": "~3.0|~4.0|~5.0",
-        "phpunit/phpunit": "^4.8",
-        "symfony/phpunit-bridge": "^3.3.8",
-        "phpspec/prophecy": "^1.7.2"
+        "symfony/yaml": "~3.0|~4.0|~5.0|~6.0",
```

Comment:
Why have you allowed packages symfony 6 for dev dependencies, but not for normal ones?

Reply:
yup, done